src/pages/plan/rules/AddEditRule/translation.ts

```diff
@@ -158,7 +158,7 @@ const defaultValues: WorkingState = {
   },
 
   lowvalue: "",
-  value: "", // input=number is a pain for users
+  value: "-5", // input=number is a pain for users
```

Comment:
Users do negative rules (expenses) more often; prefill to speed up typing and avoid forgetting `-`